api/database/migrations/2022_03_03_231414_update_users_table_with_profile_data.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateUsersTableWithProfileData extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            // New fields
+            $table->string('job_looking_status')->nullable();
+            $table->string('current_province')->nullable();
+            $table->string('current_city')->nullable();
+            $table->boolean('looking_for_english')->nullable();
+            $table->boolean('looking_for_french')->nullable();
+            $table->boolean('looking_for_bilingual')->nullable();
+            $table->string('bilingual_evaluation')->nullable();
+            $table->string('comprehension_level')->nullable();
+            $table->string('written_level')->nullable();
+            $table->string('verbal_level')->nullable();
+            $table->string('estimated_language_ability')->nullable();
+            $table->boolean('is_gov_employee')->nullable();
+            $table->boolean('interested_in_later_or_secondment')->nullable();
+            $table->uuid('current_classification')->nullable();
+            $table->foreign('current_classification')->references('id')->on('classifications');
+            $table->jsonb('location_exemptions')->nullable();
+            $table->boolean('would_accept_temporary')->nullable();
+
+            // Old fields from poolCandidate
+            $table->boolean('is_woman')->nullable();
+            $table->boolean('has_disability')->nullable();
+            $table->boolean('is_indigenous')->nullable();
+            $table->boolean('is_visible_minority')->nullable();
+
+            $table->boolean('has_diploma')->nullable();
+            $table->string('language_ability')->nullable();
+            $table->jsonb('location_preferences')->nullable();
+            $table->jsonb('expected_salary')->nullable();
+        });
+
+        // New tables to replace connections with pool candidates
+        Schema::create('operational_requirement_user', function (Blueprint $table) {
+            $table->uuid('id')->primary('id');
+            $table->timestamps();
```

Comment:
I noticed that the timestamps in the pivot tables aren't working.  This isn't really your problem since it seems they weren't working with pool candidate either.  :grin:  Could you either fix this or open a new issue for it?